research/audioset/yamnet/features.py

```diff
@@ -54,14 +54,13 @@ def waveform_to_log_mel_spectrogram(waveform, params):
     return log_mel_spectrogram
 
 
-def spectrogram_to_patches(spectrogram, params):
-  """Break up a spectrogram into a stack of fixed-size patches."""
+def log_mel_spectrogram_to_patches(spectrogram, params):
```

Comment:
I'm torn because there's nothing about this code that limits it to *log-mel* spectrograms, it just slices a tall array into patches.  But then it should be called tall_array_to_patches.  I understand that log_mel_spectrogram... avoids misdirecting how way the code reads when it's used, but I think my difference is that for me a log-mel spectrogram is still a spectrogram; "spectrogram" refers to a wide class of time-frequency representations.  I'm just responding to what feels like an implied rebuke to the previous name, I think this is fine.

Reply:
Merged both functions since we never call framing separately anyway, and the remaining single function can now unambiguously use log_mel_spectrogram.